index.html

```diff
@@ -419,11 +424,12 @@ <h3>BrowserCaptureMediaStreamTrack extension</h3>
                     <p>
                       Call the track's state before this method invocation <var>PRE-STATE</var>, and
                       after this method invocation <var>POST-STATE</var>. The user agent MUST
-                      resolve <var>p</var> when it is guaranteed that no more frames [=restricted=]
+                      <a data-cite="!HTML#queue-a-global-task">queue a global task</a> to resolve
+                      <var>p</var> when it is guaranteed that no more frames [=restricted=]
                       (or [=unrestricted=]) according to <var>PRE-STATE</var> will be delivered to
                       the application, and that any additional frames delivered to the application
-                      will therefore be [=restricted=] (or [=unrestricted=]) according to either
-                      <var>POST-STATE</var> or a later state.
+                      will therefore be [=restricted=] (or [=unrestricted=]) according to
```

Comment:
nit: I suspect my [own formatter](https://github.com/screen-share/element-capture/blob/main/.prettierrc.json) will undo this change next time I commit.

Reply:
Rewrapped